modelci/experimental/finetuner/trainer.py

```diff
@@ -93,12 +93,12 @@ def __init__(
     def from_training_job(cls, training_job: TrainingJob) -> 'PyTorchTrainer':
         # TODO: only support fine-tune
 
-        model_bo = ModelService.get_model_by_id(training_job.model)
-        if model_bo.engine != Engine.PYTORCH:
-            raise ValueError(f'Model engine expected `{Engine.PYTORCH}`, but got {model_bo.engine}.')
+        cls.model_bo = ModelService.get_model_by_id(training_job.model)
```

Comment:
you shouldn't do something like this. As we have stored the model ID, you should use this model ID (by calling `self.model`) to retrieve the model object.

Reply:
The model ID hasn't been stored,  as `self.model` is `pl.LightningModule` object which stores the net instead of the model id

Reply:
> The model ID hasn't been stored, as `self.model` is `pl.LightningModule` object which stores the net instead of the model ID

Sorry, I just find out that the model ID already is saved in `self.trainer_kwargs`

Reply:
you can add an attribute `model_id` to Trainer